Changelogs for Digital Twin Manager Plugins 6.0.1 (#4547)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1025-0-1-display-unified-delete-option-for-location-property-ctm-2843.md

---
date: ""

> title: "Display unified delete option for Location property [CTM-2843]"

Reviewer: ```suggestion
title: "Display unified delete option for Location property"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2843"
version: "1025.0.1"
---

> Previously, the “Remove” button appeared twice: one for the fields and

Reviewer: ```suggestion
In the **Add asset** view, previously, two remove buttons appeared, one for the fields and
```

> Previously, the “Remove” button appeared twice: one for the fields and

Reviewer: Is a "remove" or "delete"? Title says delete, text says remove

Reviewer 2: It is a remove button

> one for the map when the Location property was selected for the models

Reviewer: ```suggestion
one for the map when the Location property was selected.
```

> This

Reviewer: ```suggestion
```

> update ensures that only a single remove button appears for the

Reviewer: ```suggestion
This update ensures that only a single remove button appears for the property.
```

> that allowed to add additional properties in new Add asset screen. This

Reviewer: ```suggestion
This
```

> property.

Reviewer: ```suggestion
```
